File: RoRClient/Commands/LoadMapCommand.cs
using RoRClient.Commands.Base;
using RoRClient.Communication.DataTransferObject;
using RoRClient.Models.Session;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RoRClient.Commands
{
    class LoadMapCommand : CommandBase
    {

        public LoadMapCommand(RoRSession session, MessageInformation message) : base(session, message)
        {

        }


        public override void Execute()
        {

        }
    }
}

File: RoRServer/src/main/java/commands/game/LoadMapCommand.java
package commands.game;

import commands.base.CommandBase;
import communication.MessageInformation;
import models.session.RoRSession;

public class LoadMapCommand extends CommandBase {

	public LoadMapCommand(RoRSession session, MessageInformation messageInfo) {
		super(session, messageInfo);

	}

	@Override
	public void execute() {
		System.out.print("Ich soll eine DefaultMap laden!");
	}

}
